shared_test_utils/integration_test_utils.py

```diff
@@ -0,0 +1,308 @@
+#!/usr/bin/env python3
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+
+from pathlib import Path
+from typing import Optional
+from integration_constatns import *
+
+def find_charm(charm_dir: str, charm_file_name: str) -> Optional[str]:
+    """Locates a specific charm.
+
+        Args:
+            charm_dir (str): Charm dir to search in
+            charm_file_name (str)
+
+        Returns:
+            str: path to charm
+        """
+    try:
+        path = next(Path(charm_dir).rglob(charm_file_name))
+        return str(path)
+    except StopIteration:
+        return None
+
+
+async def deploy_postgresql(ops_test):
```

Comment:
We should add a parameter for the channel. For freezing at release time, and right now we're deploying from `14/stable` for using the new postgres lib.
```suggestion
async def deploy_postgresql(ops_test, channel="edge"):
```

Reply:
And we might want to do so for every charm, for the time of the release it would be handy.

We coud have a dict/yaml mapping charms to channels, used for populating both the bundle and itests. This way at release time, it would be only a matter of modifying this file. Just a quick idea.

Reply:
We don't need to specify any channel for `orc8r` charms because they are always built from the current branch, in any case, release branch or not.
For external charms, when no channel is specified, then `latest/stable` is used, which is the case with releases `1.8` and `1.6`, I updated the utils file to match that behaviour.
Done.